app/views/dashboard/autoyast.xml.erb

```diff
@@ -40,6 +40,32 @@
       </script>
 <% end %>
     </chroot-scripts>
+<% if @proxy_systemwide %>
+    <postpartitioning-scripts config:type="list">
```

Comment:
Why postpartitioning-scripts instead of chroot-scripts? If proxies are normally done in stage 2 (post reboot), then why would we do them so early? Will yast fix the permissions on the 2 folder we create? /root needing to being go-rwx and /etc/sysconfig being go+rx